cmd/keys/export/export.go

```diff
@@ -0,0 +1,57 @@
+package export
+
+import (
+	"fmt"
+	"github.com/dymensionxyz/roller/cmd/consts"
+	"github.com/dymensionxyz/roller/cmd/utils"
+	"github.com/spf13/cobra"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func Cmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export <key-id>",
+		Short: "Exports the private key of the given key id",
+		Run: func(cmd *cobra.Command, args []string) {
+			home := cmd.Flag(utils.FlagNames.Home).Value.String()
+			config, err := utils.LoadConfigFromTOML(home)
+			utils.PrettifyErrorIfExists(err)
+			supportedKeys := []string{
+				consts.KeysIds.HubSequencer,
+				consts.KeysIds.RollappSequencer,
+			}
+			keyID := args[0]
+			if keyID == consts.KeysIds.HubSequencer {
+				exportKeyCmd := getExportKeyCmdBinary(keyID, filepath.Join(home, consts.ConfigDirName.HubKeys),
+					consts.Executables.Dymension)
+				out, err := utils.ExecBashCommand(exportKeyCmd)
+				utils.PrettifyErrorIfExists(err)
+				fmt.Println(out.String())
+			} else if keyID == consts.KeysIds.RollappSequencer {
+				exportKeyCmd := getExportKeyCmdBinary(keyID, filepath.Join(home, consts.ConfigDirName.Rollapp),
+					config.RollappBinary)
+				out, err := utils.ExecBashCommand(exportKeyCmd)
+				utils.PrettifyErrorIfExists(err)
+				fmt.Println(out.String())
+			} else {
+				utils.PrettifyErrorIfExists(fmt.Errorf("invalid key id: %s. The supported keys are %s", keyID,
+					strings.Join(supportedKeys, ", ")))
+			}
+		},
+		Args: cobra.ExactArgs(1),
+	}
+	utils.AddGlobalFlags(cmd)
```

Comment:
u can move it the root command

Reply:
Why do you ask? I find managing the keys via the keys subcommand to be quite convenient

Reply:
I meant only the global flags

Reply:
good catch, fixed